mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/DeletedMessageVaultHook.java

```diff
@@ -135,6 +135,7 @@ public final int hashCode() {
         return mapperFactory.getMessageIdMapper(session)
             .findReactive(ImmutableList.of(deletedMessageMailboxContext.getMessageId()), MessageMapper.FetchType.Full)
             .next()
+            .switchIfEmpty(Mono.error(new RuntimeException("Cannont find " + deletedMessageMailboxContext.getMessageId())))
```

Comment:
:+1: the problem is here, but I did not understood why

Reply:
Me neither, it seems we are not able to read written data.

Reply:
Cassandra nconsistency on one node :trollface:  ?